.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
-    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Enabled": false,
     "python.linting.enabled": true,
     "python.linting.pylintEnabled": false,
+    "python.linting.mypyEnabled": true,
```

Comment:
Maybe we should set flake8Enabled to be true?
Also could you on your branch add `.vscode` into `.gitignore`? I will update main as well!